tests/playwright/sanity/core/app/modules/onboarding/onboarding.test.js

```diff
@@ -1,144 +1,181 @@
-
-	await expect( goProPopover ).toBeVisible();
+test.beforeEach( async ( { page }, testInfo ) => {
+	helper = new onboarding( page, testInfo.project.use.baseURL );
 } );
 
-/**
- * Test the first onboarding page - Test that the Action button at the bottom shows the correct "Create my account"
- * text, And that clicking on it opens the popup to create an account in my.elementor.com
- */
-test( 'Onboarding Create Account Popup Open', async ( { page } ) => {
-	await page.goto( '/wp-admin/admin.php?page=elementor-app#onboarding' );
-
-	const ctaButton = await page.waitForSelector( 'a.e-onboarding__button-action' );
-
-	await expect( await ctaButton.innerText() ).toBe( 'Create my account' );
-
-	const [ popup ] = await Promise.all( [
-		// It is important to call waitForEvent before click to set up waiting.
-		page.waitForEvent( 'popup' ),
-		// Opens popup.
-		page.click( 'a.e-onboarding__button-action' ),
-	] );
-
-	await popup.waitForLoadState( 'domcontentloaded' );
-
-	const createAccount = await popup.locator( 'text=Sign up for Elementor' );
+test.beforeAll( async ( { browser }, testInfo ) => {
+		const page = await browser.newPage(),
+		helper = new onboarding( page, testInfo.project.use.baseURL );
 
-	// Check that the popup opens the Elementor Connect screen.
-	await expect( createAccount ).toBeVisible();
-
-	popup.close();
+		await helper.gotoThemesPage();
+		await page.waitForLoadState( 'networkidle' );
+		await helper.activateTwenty21Theme();
 } );
 
-/**
- * Test the "Skip" button - to make sure it skips to the next step.
- */
-test( 'Onboarding Skip to Hello Theme Page', async ( { page } ) => {
-	await page.goto( '/wp-admin/admin.php?page=elementor-app#onboarding' );
-
-	const skipButton = await page.waitForSelector( 'text=Skip' );
-
-	await skipButton.click();
-
-	const PageTitle = await page.waitForSelector( '.e-onboarding__page-content-section-title' ),
-		pageTitleText = await PageTitle.innerText();
-
-	// Check that the screen changed to the Hello page.
-	await expect( pageTitleText ).toBe( 'Every site starts with a theme.' );
+test.describe( 'First Step - Elementor Account', () => {
+	test( '"Upgrade" CTA Works and Check for Broken CSS', async ( { page } ) => {
+		await helper.checkBrokenCSS( page, helper.step1URL );
+		await helper.hoverOverGoProHeaderButton();
+		await helper.checkGoProPopoverIsVisible();
+		await helper.checkUpgradeFeatureListIsCorrect( testData.upgradeFeaturesList );
+		await helper.openPopUpToProPage();
+	} );
+
+	test( '"Already Have Elementor Pro" link on popover works', async ( { page } ) => {
+		await helper.gotoStep1();
+		await helper.hoverOverGoProHeaderButton();
+
+		const [ importPopup ] = await Promise.all( [
+			page.waitForEvent( 'popup' ),
+			helper.selectAlreadyHaveElementorProLink(),
+		] );
+
+		await expect( await importPopup.url() ).toContain( 'uploadAndInstallPro' );
+	} );
+
+	test( '"Connect Your Account" link works', async ( { page } ) => {
```

Comment:
If you don't use `page`, see if you can just remove it from the test (in all places)